app/src/sharedTest/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityTest.kt

```diff
@@ -82,6 +87,53 @@ class AdministratorControlsActivityTest {
     }
   }
 
+  @Test
+  fun testAdministratorControlsFragment_loadFragment_displayGeneralAndProfileManagement() {
```

Comment:
Test cases are failing because of new implemenation.

Reply:
Done